src/main/java/com/hms_networks/sc/cumulocity/api/CConnectorApiMessageReader.java

```diff
@@ -425,13 +425,27 @@ public static void runCommand_511(
       command = command.substring(1, command.length() - 1);
     }
 
+    List topicParts = StringUtils.split(mqttTopic, "/");
+    String childDeviceName;
+    if (topicParts.size() > 2) {
+      childDeviceName = (String) topicParts.get(2);
+    } else {
+      childDeviceName = null;
+    }
+
     // Sp1lit command to parts
```

Comment:
is that supposed to be split?

Reply:
This is out of the scope of this PR, but I added a commit to address.